crates/environ/src/lib.rs

```diff
@@ -87,7 +86,26 @@ macro_rules! for_each_libcall {
             (CeilF64, wasmtime_f64_ceil),
             (FloorF64, wasmtime_f64_floor),
             (TruncF64, wasmtime_f64_trunc),
-            (NearestF64, wasmtime_f64_nearest)
+            (NearestF64, wasmtime_f64_nearest),
+            (Probestack, wasmtime_probestack),
```

Comment:
Could this actually get removed since I don't think it's used anywhere any more?

Reply:
FWIW I'm getting `Libcall::Probestack` relocation entries in large code base. (I'll find out where exactly it is coming from.)